.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+lib
```

Comment:
Any reason not to keep the compiled code? I feel like it could be easier to debug a tagged version if the compiled code is saved.